lib/index.js

```diff
@@ -159,7 +175,14 @@ function getPolyfillString(options) {
 					}
 				});
 				if (builtPolyfillString) {
-
+					return streamToString(
+						browserify(intoStream(builtPolyfillString))
+						.transform({global: true}, 'uglifyify')
```

Comment:
We could look to using [rollup](https://github.com/rollup/rollup) when requesting `.min.js`